elasticsearch/sgconfig/sg_config.yml

```diff
@@ -21,8 +21,9 @@ searchguard:
                  verb: get
                  resource: /metrics
                jaeger:
-                 verb: manage
-                 resource: /jaeger
+                 verb: get
+                 resource: jaeger
```

Comment:
why isn't this `/jaeger` like `/metrics` above?

Reply:
It does not seem to make any difference. I will change it back to `/jaeger`